sql/dql/what-is-sql/what-is-a-database-server.md

```diff
@@ -37,7 +37,7 @@ When referring to the *software*, a database server is the program that receives
 ---
 ## Practice
 
-The software on which information in a database is *stored* is called:
```

Comment:
I kinda think this was more accurate before. with the re-write, I'd answer "database"

Reply:
comments suggested that you can't store information on a software
![image](https://user-images.githubusercontent.com/19802557/62142823-e2df0680-b2e6-11e9-901c-8ea7c08890a9.png)

Reply:
"in which"?

Reply:
Flip the question from location-based to purpose-based.

> The software responsible for **storing** the database information is:

Reply:
fixed